torch/_prims/__init__.py

```diff
@@ -1661,7 +1669,7 @@ def _split_dim_aten(a: Tensor, dim: int, outer_length: int) -> Tensor:
 
 # TODO: consider renaming split_dim_view
 split_dim = _make_prim(
-    schema="split_dim(Tensor(a) a, int dim, SymInt outer_length) -> Tensor(a)",
+    schema="split_dim(Tensor(a) a, SymInt[] original_shape, int dim, SymInt outer_length) -> Tensor(a)",
```

Comment:
should we remove split_dim primitive and instead make it a ref/utility? The original motivation behind split_dim is that it's a less general operation than `view` and thus can be easier handled by compiler, but if in the end all these ops are sent to nvfuser's view, there's no point in having them as the different prims. cc @mruberry

Reply:
Currently, it's sent to nvfuser's existing view, implemented for TorchScript support, because it's the easiest approach at the moment. I will work on changing that to a less general split_dim function, that doesn't exist yet, but is not difficult to add.